ea_dicm2nii.m

```diff
@@ -1,3 +1,17 @@
 function ea_dicm2nii(indir,outdir)
+%
+%
+% USAGE:
+%
+%    ea_dicm2nii(indir,outdir)
+%
+% INPUTS:
```

Comment:
Please provide a description of the inputs.

Reply:
```
%    indir: DICOM folder
%    outdir: output folder for NIfTI files
```